python/printing_horizontally.md

```diff
@@ -0,0 +1,19 @@
+# Printing Horizontally
+
+The print() function can't be used to print horizontally (simulate a user 
+typing) because print() inserts a new line at the end of the string. Instead, 
+sys.stdout.write() has to be used.
+
+
+## Example
+
+    import sys
```

Comment:
Will you put this in a [fenced code block](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks) and specify the language as python?